pkgs/tools/compression/efficient-compression-tool/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, boost
+, cmake
+, nasm
+}:
+
+stdenv.mkDerivation rec {
+  pname = "efficient-compression-tool";
+  version = "0.8.3";
+
+  src = fetchFromGitHub {
+    owner = "fhanau";
+    repo = "Efficient-Compression-Tool";
+    rev = "v${version}";
+    sha256 = "sha256-IHqIKVUvis9LtxFAQz7ItBHZ+BxxWD1VKiNmlTV+UyI=";
+    fetchSubmodules = true;
```

Comment:
```suggestion
```
Please use our libpng.

Reply:
I am not sure this is possible: the package uses a customised build of `libpng` (see `pngusr.h`).

Reply:
It basically disabled support for a lot of things and set those 3 values https://github.com/fhanau/Efficient-Compression-Tool/blob/master/src/pngusr.h#L92-L94

We could just overwrite our libpng. The libpng version used here is 3 years old.